src/main/java/graphql/language/AbstractNode.java

```diff
@@ -56,15 +59,15 @@ public Map<String, String> getAdditionalData() {
     }
 
     @SuppressWarnings("unchecked")
-    protected <V extends Node> V deepCopy(V nullableObj) {
+    protected <V extends @Nullable Node> @Nullable V deepCopy(@Nullable V nullableObj) {
```

Comment:
I don't understand this change @dondonz ... why should we allow null values here?

Reply:
The first line we check whether the input is nullable `if (nullableObj == null) {` so I have allowed the input to be nullable

I agree it's strange in the context of `Value` like `FloatValue`. 

Maybe there was another reason to allow this to be nullable in the first place?

Reply:
So we have two options here: either declare `V` as a type that also can be null or declare input and output as Nullable. We have done both here, which doesn't make sense I think.